pom.xml

```diff
@@ -45,25 +45,10 @@
         <sonar.organization>navit</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <nav.security.version>1.3.3</nav.security.version>
-        <!--suppress UnresolvedMavenProperty Ligger som secret i github-->
-        <spring.cloud>3.0.1</spring.cloud>
-        <spring.cloud-contract>3.0.1</spring.cloud-contract>
         <!--suppress UnresolvedMavenProperty -->
         <sonar.login>${SONAR_LOGIN}</sonar.login>
     </properties>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
```

Comment:
Trengs ikke spring cloud eller kommer det via-via?

Reply:
Slik jeg har tolket det har cloud kun blitt brukt for håndtering av vault og database og nå kommer vi bruke secrets fra kubernetes(google) og env-variabler fra nais som legger inn database-username/pwd